e2e/docker-compose.yml

```diff
@@ -45,3 +47,9 @@ services:
       POSTGRES_DB: immich
     ports:
       - 5435:5432
+    healthcheck:
```

Comment:
I believe this part isn't needed since it is specified in the dockerfile itself for the database, but otherwise looks good

Reply:
I see this is a smaller interval which seems fine for this purpose